src/shell/shell_cmds.cpp

```diff
@@ -418,23 +418,29 @@ void DOS_Shell::CMD_ECHO(char * args){
 	} else WriteOut("%s\r\n",args);
 }
 
-int64_t ticks_at_program_launch = 0;
 void DOS_Shell::CMD_EXIT(char *args)
 {
 	HELP("EXIT");
 
 	const bool wants_force_exit = control->cmdline->FindExist("-exit");
-	const bool is_normal_launch = control->GetStartupVerbosity() != Verbosity::InstantLaunch;
-	const auto seconds_since_launch = GetTicksSince(ticks_at_program_launch) / 1000.0;
+	const bool is_normal_launch = control->GetStartupVerbosity() !=
+	                              Verbosity::InstantLaunch;
 
-	if (wants_force_exit || is_normal_launch || seconds_since_launch > 1.5) {
+	// Check if this is an early-exit situation, in which case we avoid
+	// exiting because the user might have a configuration problem and we
+	// should let them see any errors in their console.
+	constexpr auto early_exit_s = 1.5;
+	const auto exiting_after_s  = DOSBOX_GetUptime();
```

Comment:
Perhaps names like `early_exit_seconds` and `existing_after_seconds` are more readable than `early_exit_s` and `existing_after_s`?

Reply:
Updated, thanks!